keyvi/src/cpp/dictionary/fsa/internal/sparse_array_persistence.h

```diff
@@ -78,8 +78,8 @@ final {
           "dictionary-fsa-%%%%-%%%%-%%%%-%%%%");
       boost::filesystem::create_directory(temporary_directory_);
 
-      // size of external memory chunk: not more than 1 or 4 GB
-      size_t external_memory_chunk_size = std::min(flush_size_ * 10, (size_t)1073741824);
+      // size of external memory chunk: not more than 1 GB
+      size_t external_memory_chunk_size = std::min(flush_size_, (size_t)1073741824);
```

Comment:
I do not think this is a good idea. You should rather lower the memory limit, you still configure 256 MB there, that's really not necessary for some small unit tests.